neqo-transport/src/flow_mgr.rs

```diff
@@ -51,10 +51,16 @@ impl FlowMgr {
         assert!(self.used_data <= self.max_data)
     }
 
+    // Dummy DataBlocked frame for discriminant use below
+    const DB_FRAME: Frame = Frame::DataBlocked { data_limit: 0 };
```

Comment:
Can't you just defined this at the point it is used?  I didn't imagine that this would need to be moved to an outer scope.